README.rst

```diff
@@ -523,8 +525,6 @@ Choose from the list of available configurations:
 The ``--dry-run`` option displays the files that need to be
 fixed but without actually modifying them:
 
-.. code-block:: bash
```

Comment:
I'm running the `php php-cs-fixer readme > README.rst` script. Why does these get removed?